docs/development/methodology/trello.md

```diff
@@ -39,7 +39,7 @@ cards is blue.
 
 ### Use of Trello in Weekly Meetings
 
-Trello is the main way in which we can communicate with the client
+Trello is the main way in which we communicate to the client
```

Comment:
When I see "to" I think in communicate <something> to the client. By just putting communicate to the client I feel like there's something missing to communicate.